### ISSUE TITLE
Exosuit Fabricator treats all stacks of materials as a stack with as much sheets as the Fabricator can hold.

### ISSUE THREAD

**Issue by GreyJustice1** [vgstation-coders/vgstation13#826]
How to reproduce the bug:
1. Pick a stack of materials (e.g. metal) with any number of sheets
2. Put it in Exosuit Fabricator
3. "You insert 70 metal into the fabricator."
It doesn't matter if the stack had 1 sheet or 50 sheet.

Also, if you remove all of the materials after you have done this, you get a stack with more than 50 sheets.
I have tested this with almost all materials with the default Exosuit Fabricators.

If the exosuit fabricator has better matter bins, even more materials get added.

Basically, you can make infinite amount of any material the Exosuit Fabricators can hold with only one sheet of the material.
